src/hdmf/container.py

```diff
@@ -187,13 +188,17 @@ def __new__(cls, *args, **kwargs):
         inst.parent = kwargs.pop('parent', None)
         return inst
 
-    @docval({'name': 'name', 'type': str, 'doc': 'the name of this container'})
+    @docval({'name': 'name', 'type': str, 'doc': 'the name of this container'},
+            {'name': 'ontology', 'type': Ontology,
+             'doc': 'Optional ontology solely for thr purpose of controlling data values',
+             'default': None})
     def __init__(self, **kwargs):
         name = getargs('name', kwargs)
         if '/' in name:
             raise ValueError("name '" + name + "' cannot contain '/'")
         self.__name = name
         self.__field_values = dict()
+        self.ontology = kwargs['ontology']
```

Comment:
It does not make sense to have an ontology on a container. Which attribute or dataset of the container is being associated with this ontology? I think it does make sense to have it as a property of Data though. So I would recommend moving this there.